deckhouse-controller/pkg/controller/deckhouse-release/check_release.go

```diff
@@ -559,7 +573,7 @@ func (dcr *DeckhouseReleaseChecker) nextVersion(actual, target *semver.Version)
 	}
 
 	collection := make([]*semver.Version, 0)
-	for _, ver := range listTags {
+	for _, ver := range tags {
 		if r.MatchString(ver) {
 			newSemver, err := semver.NewVersion(ver)
 			if err != nil {
```

Comment:
We set a regexp to find the version. And if we ended up here, it means the version matches our regexp. And that means there can't be an error parsing the version.
But that's just thinking out loud.